mast_aladin_lite/aida.py

```diff
@@ -1,4 +1,5 @@
 from astropy.coordinates import SkyCoord
+from typing import Any
```

Comment:
`jdaviz` doesn't use typing, and I'd advocate for avoiding it in mast-aladin-lite. Happy to hear differing opinions.
```suggestion
```